src/java/hillel/elementary/homework6/Main.java

```diff
@@ -0,0 +1,47 @@
+package hillel.elementary.homework6;
+
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) {
+
+        InitializationData initializationData = new InitializationData();
+        Information information = initializationData.fillTheData();
+
+        List<Integer> arrayList = new ArrayList<>();
+        arrayList.add(78);
+        arrayList.add(3);
+        arrayList.add(8);
+
+        Information newInfo = additionStrings(arrayList, information);
+
+        System.out.println("Email set size: " + newInfo.getEmailSet().size());
+        System.out.println("Email-Nickname map size: " + newInfo.getEmailNicknameMap().size());
+        System.out.println("Nickname list size " + newInfo.getNicknameList().size());
+    }
+
+    public static Information additionStrings(List<? extends Number> intList, Information info) {
```

Comment:
It is ok, but it will be better to change list naming (for example 'numbersList'), because this method works not only with Integer. It will be confusing if someone will try to read how it works